### ISSUE TITLE
Hollowing a corpse maintains its brain?

### ISSUE THREAD

**Issue by honeyed-lemons** [impstation/imp-station-14#1142]
After a changeling hollows out a corpse, the brain remains. I'm not sure if this is an issue or intentional, but it should not be occurring imo

**Comment by jtfg-dev** [comment]
this is "intentional game design" iirc but after looking for an hour yesterday i could not for the life of me find where this (removing certain organs from a hollowed corpse) is in the code. chalk it up to my lack of familiarity with playing changelings i guess. but this should be a one-line change

**Comment by mqole** [comment]
honestly, im okay with this being intentional game design. brains are uniquely indestructible, and mmi gaming should probably not be completely inaccessible to someone just bc they got ate

**Comment by honeyed-lemons** [comment]
yeah after playing a bit more with changelings i think i like this actually
